### ISSUE TITLE
A beginner question. but an important one

### ISSUE THREAD

**Issue by DnaLunchie** [endless-sky/endless-sky#3673]
Hey guys

I just started playing this game about 12 hours ago, am just at that stage of trading in my bounder and stealth bomber looking ship for a proper capital ship (the excitement of choosing which ship to buy next is positively amazing (Im sure you all remember that feeling).

Anyway to cut to the chase I have been reading some of the forum articles and I noticed that several ships or species you talk about are not familiar to me.

I have explored what I THINK is the whole galaxy however I have done VERY little in what i would call missions, mostly just trading and running passengers. So basically im just wanting to confirm that I just have actually got the right and full/latest version of the game. I downloaded my copy from steam about a week ago so im hoping that is the right copy? and therefore im guessing I haven't met these other ships or areas because I haven't done any real "story-line content" yet?

Basically am I on the right track? I dont want to be putting in hours and hours and find out I am playing an old version lacking  whole campaigns etc

**Comment by Brick63** [comment]
there's basically a hard lock on what you can do in the game either prior to finishing the story or getting a certain outfit some way else. so i would suggest doing the story first then try exploring some more.

**Comment by Amazinite** [comment]
The ES Github is for feature requests and bug reports. If you're wanting to ask a question like this, you should do so over on either the [Steam forums](http://steamcommunity.com/app/404410/discussions/0/) or [Google Group](https://groups.google.com/forum/#!forum/endless-sky).

To answer your question here, you need to go to the south-west of human space where the Free Worlds systems are and click spaceport whenever you land on a FW planet.

**Comment by MessyMix** [comment]
Hey DNA,

Github isn't really the place for gameplay advice—normally it's used for bug reporting, feature requests, and pull requests. Please try the Steam forums or the Google Groups instead.

You definitely have the full version of the game _as meant for public consumption_, and the reason that you haven't met these species is both because you haven't done the storyline and because you haven't been looking hard enough. You'll have the chance to meet at least 3 non-human species before you start the storyline.

**Comment by DnaLunchie** [comment]
Hey guys, sorry is/was a late night post. I had read a fair few comments on steam along the lines of "oh man the dev's have abandoned this game" and as such I figured this was the best place to ask. My question was not about what I needed to do in order to meet new parts of the game but rather simply providing a context to where I am in the game in order for you experts to correctly judge whether I have in fact been playing the correct version. I definitely don't want any gameplay hints/tips as I tend to be someone who investigates every little aspect of each zone before I move on. Regardless apoligies if I posted this incorrectly but I didn't even know about the google group and didn't want to join in the often uninformed steam bandwagon of comments..... Thanks heaps for the quick reply. Loving the game. Takes me back to 20 years ago playing Escape Velocity on my friends old apple
